modules/nw-operator-cr.adoc

```diff
@@ -206,7 +206,7 @@ endif::operator[]
 
 |`gatewayConfig`
 |`object`
-|Optional: Specify a configuration object for customizing how egress traffic is sent to the node gateway.
+|Optional: Specify a configuration object for customizing how egress traffic is sent to the node gateway. Possible values are `Shared` and `Local`. The default value is `Shared`. In the default setting, the Open vSwitch (OVS) outputs traffic directly to the node IP interface, In the `Local` setting, it traverses the host network; consequently, it gets applied to the routing table of the host.
```

Comment:
Just a suggestion to consider, but maybe switch "possible" for "valid"? Unless there's a reason to keep it as is
```suggestion
|Optional: Specify a configuration object for customizing how egress traffic is sent to the node gateway. Valid values are `Shared` and `Local`. The default value is `Shared`. In the default setting, the Open vSwitch (OVS) outputs traffic directly to the node IP interface, In the `Local` setting, it traverses the host network; consequently, it gets applied to the routing table of the host.
```